craft_parts/packages/deb.py

```diff
@@ -500,8 +506,8 @@ def unpack_stage_packages(
                 mark_origin_stage_package(extract_dir, marked_name)
                 # Stage files to install_dir.
                 file_utils.link_or_copy_tree(extract_dir, install_path.as_posix())
-        # XXX: normalization of stage packages done by the application
-        # cls.normalize(str(install_path))
+
+        normalize(str(install_path), repository=cls)
```

Comment:
`normalize` should also handle Pathlib. Otherwise we'd have half of the code with string paths, the other half with pathlib.Path, and that's no good.

Reply:
Refactored the normalize module to use pathlib.